.github/workflows/ci.yaml

```diff
@@ -11,6 +11,7 @@ on:
 env:
   RUNC_VERSION: v1.0.2
   GO_VERSION: 1.17.2
+  K8S_VERSION: 1.20.2
```

Comment:
let's find out why the latest does not work and add a comment here

Reply:
#760